packages/server/graphql/types/RetroDiscussStage.ts

```diff
@@ -2,10 +2,24 @@ import {GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType} from 'graphq
 import {NewMeetingPhaseTypeEnum} from '../../database/types/GenericMeetingPhase'
 import ReflectionGroup from '../../database/types/ReflectionGroup'
 import {GQLContext} from '../graphql'
+import Discussion from './Discussion'
 import DiscussionThreadStage, {discussionThreadStageFields} from './DiscussionThreadStage'
 import NewMeetingStage, {newMeetingStageFields} from './NewMeetingStage'
 import RetroReflectionGroup from './RetroReflectionGroup'
 
+const createEmptyDiscussion = (id) => ({
```

Comment:
the ID being `''` feels a little weird. could we just hardcode in a value like `id: 'dummy'`?